openlibrary/plugins/importapi/code.py

```diff
@@ -459,7 +459,10 @@ def login(self, authstring):
         if not authstring:
             return
         authstring = authstring.replace("Basic ","")
-        username, password = base64.decodestring(authstring).split(':')
+        if PY2:
+            username, password = base64.decodestring(authstring).split(':')
+        else:
+            username, password = base64.decodebytes(bytes(authstring, 'utf-8')).encode('utf-8').split(':')
```

Comment:
```suggestion
        try:
            username, password = base64.decodebytes(bytes(authstring, 'utf-8')).encode('utf-8').split(':')
        except AttributeError
            username, password = base64.decodestring(authstring).split(':')
```
Follows Python porting best practice [use feature detection instead of version detection](Follows Python porting best practice
https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection)

Same below and the six import is not needed.

Reply:
Also, please wrap the lines to 88 chars max.

Reply:
@cclauss I have addressed both the changes. Do take a look, and merge if found ok.